src/cljs/nr/cardbrowser.cljs

```diff
@@ -39,8 +41,37 @@
         (.setItem js/localStorage "cards" (.stringify js/JSON (clj->js {:cards cards :version server-version}))))
       (reset! all-cards cards)
       (swap! app-state assoc :cards-loaded true)
+      (swap! app-state assoc :previous-cards (generate-previous-cards cards))
```

Comment:
If you like this on a separate line for clarity, cool. If you like code golf like me, lol, `assoc` takes multiple key value pairs, so you can merge this with the line above: `(swap! app-state assoc :cards-loaded true :previous-cards ...)`

Reply:
Would I program in clojure if I wasn't trying to optimize for loc?